src/Microsoft.Extensions.CommandLineUtils/project.json

```diff
@@ -10,7 +10,8 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true
+    "xmlDoc": true,
+    "compile": "../Microsoft.Extensions.CommandLineUtils.Sources/**/*.cs"
```

Comment:
The was created to enable a console tool "ef.exe" that has no dependencies. The same goal could be accomplished with ILMerge, but we decided a .Sources package was easier.